### PR TITLE
SP4 backport

### PR DESCRIPTION
**Pull request by lslezak**
The Travis build fails for the SP4 target (https://travis-ci.org/libyui/libyui-qt-pkg/builds/4052518790),
this PR just updates the SP4 branch to the latest version from `master`.

Added 2 extra commits:

- Fixed Travis to use the SLE12-SP4 Docker image
- Submit to SLES12-SP4

### CODE REVIEW

**Comment by jreidinger** [comment]
this looks like bad idea against packaging-rake-tasks. In general idea is that the latest libyui-rake
 or yast-rake should be able to build any older project. So it is always backward compatible. Why it does not work in this case?

**Comment by lslezak** [comment]
Because the current SP4 cannot submit to SP4, the SP4 target is missing. https://travis-ci.org/libyui/libyui-qt-pkg/builds/405251879#L374

**Comment by jreidinger** [comment]
@lslezak then correct solution is to add libyui-rake from master to SP4 devel package. and submit it there.

File: libyui-rake.gemspec [hunk 1]
```diff
@@ -41,6 +41,6 @@ end
   spec.require_path = "lib"
 
   # dependencies
-  spec.add_runtime_dependency("rake", "> 10.0", "< 12")
+  spec.add_runtime_dependency("rake", "> 10.0", "< 999")
```
Comment (jreidinger) [inline libyui-rake.gemspec]: just remove the third argument?

Comment (mvidner) [reply libyui-rake.gemspec]: then `gem build` would say
"WARNING:  open-ended dependency on rake (> 10.0) is not recommended"

**Comment by lslezak** [comment]
After some discussion we decided to remove the SP4 branch and submit the package from the `master` branch to SP4.

**Pull request closed without merging**